utopia-remix/app/routes/projects.tsx

```diff
@@ -343,13 +398,13 @@ const CategoryTrashActions = React.memo(({ projects }: { projects: ProjectWithou
 
   return (
     <>
-      <button
-        className={button({ size: 'small' })}
+      <div
+        className={button({ color: 'subtle' })}
         onClick={handleEmptyTrash}
-        disabled={projects.length === 0}
+        style={{ display: projects.length === 0 ? 'none' : 'block' }}
```

Comment:
I think you can just `return null` early so it's more obvious that it will only show when there are projects